packages/frontend/src/components/UserSettings/GithubSettingsPanel/index.tsx

```diff
@@ -18,71 +17,17 @@ import {
     IconRefresh,
     IconTrash,
 } from '@tabler/icons-react';
-import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
 import { useEffect, type FC } from 'react';
-import { lightdashApi } from '../../../api';
 import useToaster from '../../../hooks/toaster/useToaster';
 import useSearchParams from '../../../hooks/useSearchParams';
 import githubIcon from '../../../svgs/github-icon.svg';
+import {
+    useDeleteGithubInstallationMutation,
+    useGitHubRepositories,
+} from '../../common/GithubIntegration/hooks/useGithubIntegration';
 import MantineIcon from '../../common/MantineIcon';
 import { SettingsGridCard } from '../../common/Settings/SettingsCard';
 
-const getGithubRepositories = async () =>
```

Comment:
moved from the settings panel into a github `hook` folder, to be reused in the login form